### DISCUSSION TITLE
Pytorch on CPU

### DISCUSSION THREAD

**Discussion by arjunzayn** [mrdbourke/pytorch-deep-learning#151] [General]
Hi there!
i have started learning the pytorch and installed on my local system  wwhich runs on cpu 11th gen intel i5 processor . And, what are the things that am gonna miss working with the cpu version?
Thanks

**Comment by fivefishstudios** [comment]
You won't miss anything other than maybe waiting longer for training sessions. And even then, sometimes running small models on cpu is faster vs. cuda (because of time lost in memory transfer to/from computer memory to gpu memory).

**Reply by mrdbourke** [reply]
thank you for the answer! @fivefishstudios
